multimodal/src/autogluon/multimodal/optimization/lit_module.py

```diff
@@ -5,6 +5,7 @@
 import torch
 import torch.nn.functional as F
 import torchmetrics
+from pytorch_lightning.utilities import grad_norm
```

Comment:
Should we at some point consider switching to the new API (as described [here](https://github.com/Lightning-AI/lightning/discussions/16688#discussioncomment-4927515)) or should we wait until the deprecation notice is released for the old API?
```python
import lightning as L
```

Reply:
Good call. We can switch to the new namespace when it becomes more stable.